lib/models/notification.ts

```diff
@@ -97,15 +97,7 @@ export type NotificationChannelMeta = t.TypeOf<typeof NotificationChannelMeta>;
 
 // Generic Notification object
 
-export const Notification = t.intersection([
-  NotificationBase,
-  t.interface({
-    channel: t.partial({
-      [NotificationChannelEnum.EMAIL]: NotificationChannelEmail,
-      [NotificationChannelEnum.WEBHOOK]: NotificationChannelWebhook
-    })
-  })
-]);
+export const Notification = t.union([WebhookNotification, EmailNotification]);
```

Comment:
I think it made much more sense before than now

Reply:
needs some context : ) 

most of the oddities here come from the fact that we started with a "fat" notification model that contained all metadata for every channel.  now we handle only specific kinds of notifications
(ie. email *or* webhook).

so regarding "why this property is called *channels* and not *channel* ?": because renaming it, despite being more correct, would be a breaking change (a change not exposed to the external API, but will break the way we retrieve notifications object from CosmosDB).

in case we rename it,it's worth to refactor the whole notification model to adhere much more to the new requirements. for example the old way (which you state that "made much more sense") is plain wrong:

```ts
export const Notification =t.interface({
 NotificationBase
  t.interface({
    channel: t.partial({	
      [NotificationChannelEnum.EMAIL]: NotificationChannelEmail	
      [NotificationChannelEnum.WEBHOO]: NotificationChannelWebhook
    })	
  })	
 ]);
```

We cannot have both `NotificationChannelEmail` *and* `NotificationChannelWebhook` (this was possible with the old fat model), so `Notification` is infact a union type. 

I've restructured the types again a little bit to "flatten" the metadata avoiding the 
"single constant key" and using it to discriminate the tagged union (= Notification).